addons/project/static/tests/project_task_kanban_renderer_tests.js

```diff
@@ -0,0 +1,126 @@
+/** @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { getFixture, nextTick } from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+import { start } from "@mail/../tests/helpers/test_utils";
+import { registry } from "@web/core/registry";
+import { makeFakeUserService } from "@web/../tests/helpers/mock_services";
+
+let target;
+let serverData;
+
+const serviceRegistry = registry.category("services");
+
+QUnit.module("project", (hooks) => {
+    hooks.beforeEach(async () => {
+        const pyEnv = await startServer();
+        pyEnv.mockServer.models["project.project"] = {
+            fields: {
+                id: { string: "ID", type: "integer" },
+                display_name: { string: "Name", type: "char" },
+            },
```

Comment:
Is it really necessary?

Reply:
Maybe, we can let as he did. In master, he will have to convert his test in HOOT. 🙂